ivy/functional/frontends/torch/nn/functional/loss_functions.py

```diff
@@ -613,3 +613,20 @@ def triplet_margin_with_distance_loss(
     loss = ivy.maximum(dist_pos - dist_neg + ivy.array(margin), ivy.array(0.0))
 
     return reduction(loss).astype(anchor.dtype)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
+def multilabel_margin_loss(input, target, reduction="mean"):
```

Comment:
The function signature should match with the torch `multilabel_margin_loss` (https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_margin_loss.html). `size_average` and `reduce` arguments are missing here.

Reply:
Hi @hello-fri-end , I have updated the args in my latest commit as with my implementation of multilabel_margin_loss. Thanks.